### DISCUSSION TITLE
Chat syncronization with Customer-Vendor Live Chat

### DISCUSSION THREAD

**Discussion by gregkeys** [inspireui/support#5488] [New Feature/Ideas Requests]
The live chat using firebase should have a front end for the website so customers can chat from website to mobile app.

OR wcfm ultimate has its own Customer-Vendor Live Chat it would be great if that could be used as the live chat option in the app, it uses firebase but the structure is different.

having two different live chat options that dont work with each other is confusing, it would be great to have one solution that works with both the website and the apps

**Comment by Uhtredtb** [comment]
Indeed, please implement this to app.

**Comment by officialsuyogdixit** [comment]
Strange that they are not in sync. Very confusing.

**Comment by Yan-Aleman** [comment]
Up
